scripts/installer.iss

```diff
@@ -301,6 +301,7 @@ fi.InstallingMSVCRedist=Asennetaan Visual C++ 2019 Redistributable...
 it.InstallingMSVCRedist=Installazione di Visual C++ 2019 Redistributable...
 pl.InstallingMSVCRedist=Instalowanie dystrybucji Visual C++ 2019...
 uk_UA.InstallingMSVCRedist=Встановлюємо Visual C++ 2019 Redistributable...
+ja.InstallingMSVCRedist=Visual C++ 再頒布可能パッケージをインストール中...
```

Comment:
I forgot about it too for a minute 😛. It'd be good to keep these sorted alphabetically. Also, the year should be kept in the string to inform users of what MSVC Redistributable is being installed, though feel free to reformat or translate it for Japanese as you see fit. Otherwise, everything looks good to me:

```suggestion
ja.InstallingMSVCRedist=Visual C++ 2019 再頒布可能パッケージをインストール中...
pl.InstallingMSVCRedist=Instalowanie dystrybucji Visual C++ 2019...
uk_UA.InstallingMSVCRedist=Встановлюємо Visual C++ 2019 Redistributable...
```

Reply:
Looks like you just missed the 2019.